zerberus/zone.h

```diff
@@ -76,6 +76,7 @@ struct Zone {
       float ampegVel2Sustain  = 0.0;
       float ampegVel2Release  = 0.0;
       float rtDecay = 0.0;
+      float delay = 0.0;
```

Comment:
float here, double above?

Reply:
These are different variables. I'm also a bit confused by two questions: why there are doubles and floats and why don't compilator give me warnings on conversion. Will understand and replace to floats if possible.

Reply:
You use `readDouble()` so probably need doubles all the way? Or cast. Are fractional seconds needed? If not int should do

Reply:
and what is that `* 1000` about then?

Reply:
All calculations are in milliseconds, but user specifies most opcodes in seconds. Float exists because of the further calculations, interpolations, amplifications and so on.